### PR TITLE
Added package-lock.json to gitignore file and fixed the npm postinstall…

### PR DESCRIPTION
**Pull request by ReddragonLR**
Added package-lock.json to gitignore file and updated the postinstall npm script to remove everything from global installation besides the jquery as this was breaking the npm install process

### CODE REVIEW

**Comment by ReddragonLR** [comment]
CLA signed...

**Comment by Splaktar** [comment]
I am seeing the `Attempted to compile "angular-animate" as a global module, but it looks like an external module. You'll need to remove the global option to continue.` error on postinstall.

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ typings.json
 .DS_Store
 .idea
 !.gitkeep
+package-lock.json
```
Comment (Splaktar) [inline .gitignore]: The `package-lock.json` should be included in source control.

**Comment by Splaktar** [comment]
Thank you for letting us know about this issue! It's been fixed in https://github.com/angular/material-start/commit/bbd3f569925283d186b9dac9f2612e8339e70728.

**Pull request closed without merging**